File: src/minecraft/monnef/jaffas/xmas/ItemXmasFood.java
package monnef.jaffas.xmas;

import monnef.jaffas.food.item.ItemJaffaFood;

public class ItemXmasFood extends ItemJaffaFood {
    public ItemXmasFood(int id) {
        super(id);
        this.setCreativeTab(jaffasXmas.CreativeTab);
    }

    /*
    @Override
    public String getTextureFile() {
        return "/jaffas_04.png";
    }
    */
}

File: src/main/scala/monnef/jaffas/xmas/item/ItemXmasFood.scala
package monnef.jaffas.xmas.item

import monnef.jaffas.food.item.ItemJaffaFood
import monnef.jaffas.xmas.common.IconDescriptorXmas
import monnef.jaffas.xmas.JaffasXmas

class ItemXmasFood extends ItemJaffaFood with IconDescriptorXmas {
  setCreativeTab(JaffasXmas.instance.creativeTab)
}
